lib/_tls_wrap.js

```diff
@@ -1540,25 +1543,14 @@ function onConnectEnd() {
   }
 }
 
-let warnOnAllowUnauthorized = true;
-
 // Arguments: [port,] [host,] [options,] [cb]
 exports.connect = function connect(...args) {
   args = normalizeConnectArgs(args);
   let options = args[0];
   const cb = args[1];
-  const allowUnauthorized = process.env.NODE_TLS_REJECT_UNAUTHORIZED === '0';
-
-  if (allowUnauthorized && warnOnAllowUnauthorized) {
-    warnOnAllowUnauthorized = false;
-    process.emitWarning('Setting the NODE_TLS_REJECT_UNAUTHORIZED ' +
-                        'environment variable to \'0\' makes TLS connections ' +
-                        'and HTTPS requests insecure by disabling ' +
-                        'certificate verification.');
-  }
```

Comment:
This seems wholly unrelated to QUIC

Reply:
Yes and no. We use the same warning in the quic implementation so it was moved out of _tls_wrap.js to make it more generally reusable.

Reply:
I see now, it moved. worth putting in its own PR to keep this one more quick specific